src/App.tsx

```diff
@@ -1,10 +1,59 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+
+import { getTodos } from './api/todos';
+
+import { Header } from './components/ErrorMessage/Header/Header';
+import { TodoList } from './components/ErrorMessage/TodoList/TodoList';
+import { Footer } from './components/ErrorMessage/Footer/Footer';
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
+import { TodoListState } from './types/TodoListState';
+import { Errors } from './types/Errors';
+
+const USER_ID = 12018;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
```

Comment:
```suggestion
  const [todoList, setTodoList] = useState<Todo[]>([]);
```
let's just set the empty array by default